### ISSUE TITLE
ENZIP 14: Introducing an Enzyme Star Managers Program

### ISSUE THREAD

**Issue by jindouyunz** [melonproject/MIP#14]
**ENZIP: 14
Title: Introducing an Enzyme Star Managers Program
Author: Jindouyunz
Status: Draft
Type: ENZIP
Created: Feb 23, 2023**
 
**Abstract**
Enzyme is a mature (and in my opinion the best) asset management tool but has not seen mass adoption in the crypto space (in terms of TVL and users). I think one of the particularly important reasons is that there are not enough vault managers with excellent performance on the Enzyme platform. Crypto users are currently looking for profit. If Enzyme can have enough high-return vaults for them to choose from, users and assets will naturally come. This proposal will describe how to attract good vault managers through the Star Managers Program.
 
**Background**
At present, in the mainstream financial field, the scale of asset management accounts for about 24%, while crypto asset management accounts for only 0.15% (from Messari). Calculated according to the above ratio, it is entirely possible for Enzyme to become a $1 billion project, and Enzyme's current TVL is $60 million. What prevents Enzyme from becoming a $1 billion TVL DEFI protocol? I think it's a combination of the following reasons:
1.	Segment timing: Finance follows demand. From the perspective of the history of mainstream finance, lending and trading are segments that emerged earlier and grew up. They correspond to the most basic financial needs of people. People need to borrow money to start a business and trade commodities. The large-scale development of asset management appears to be later. It needs business development to a certain level before there are enough investment opportunities. It relies on the prosperity of securities and other markets, providing convenient exit channels, professional investors, etc. The same is true for Crypto, so we see that the current members of the $1 billion club meet these most basic needs: Uniswap, Curve, Maker, Aave, and Compound. The 2020 DEFI summer has made these protocols at the bottom of the demand pyramid flourish.
Nowadays, relatively stable annual return ratios such as income optimization and Liquidity Mining have entered the era of single digits, and people will start to seek higher-yield opportunities. It is only a matter of time before the growth and explosion of the asset management segment. Optimistic estimates require 1 more super cycle, and pessimistic estimates require 2 or 3 more cycles. → At this point, what Enzyme needs to do is to make all the preparations when the wind comes.
2.	There are not enough excellent vault managers: The logic is simple. Many vaults were established in 2020-2021. Now the market has gone through a complete supercycle, but vault managers have not created a correspondingly high return (Maybe they created it, but they didn’t grasp the risks when the market went down. Of course, this cannot be used as an excuse, because risk management is part of an investment.) Many people can’t find a suitable vault to invest in, so TVL can’t grow. 
Below is the actual data collected from the Enzyme platform.
https://docs.google.com/spreadsheets/d/1Lk-MQ72trTsHqpoYubmukoZZ5aUehxjPGTCE-zJmDbI/edit?usp=sharing.
I list the returns since their inception of the top 30 vaults by AUM. Thanks to @Ignacio for pointing out that this sample is biased and does not reflect the level of the best-performing vaults. I agree. I do this because I can't easily sort the return since inception from the website, and I don't know which vaults are the best performers (I think many investors should have the same confusion as me). Manually counting the data of 1200+ vaults is also time-consuming. But I still think this biased data is valuable. Although it is a small sample, but not a bad sample. If the return is high, the AUM will also increase, and it is more likely to become a vault with top AUM. Of course, I hope that someone can make statistics on the overall 1200+, so that the conclusions drawn will be more accurate.
I think the above data can support my second point of view to a certain extent: no vaults with high returns → no depositor and TVL. The optimistic point is that, compared to the first macroscopic problem, the second problem can be solved by Enzyme.
 
**Star Managers Program Overview**
There is no shortage of outstanding investment masters in this market. They may have become multi-millionaires, or Twitter KOLs, or are still accumulating their own funds. But why are they unwilling to come to the Enzyme platform to display their talents and get greater returns? I think, first of all, they may not know that there is such a platform as Enzyme. Similarly, we have many investors who want to gain wealth in the Crypto field. They have spare money, but their own investment level is not high, and they cannot enter the top VC with a high threshold.
So the Star Managers Program will be divided into two parts:
1.	Star Managers Contest: Through competitions and high bonuses, attract investment masters in the market to the Enzyme platform and become our star managers
2.	Star Managers Campaign: Promote these star managers to the market and potential investors to attract investors and funds
 
**Star Managers Contest-------**
Star Manager Contest is a manager investment competition, which attracts talented investors/traders in the crypto field to create and operate vaults on the Enzyme platform through high bonuses. At the same time play the role of marketing.
**Contestants**: open, permissionless
**How to enter**: Create a vault on Enzyme, or use an existing vault
**Bonuses**: issued in $MLN, the total bonus scale is 50,000 $MLN (but up to $500,000). $MLN is newly minted, and the undistributed part of the minted $MLN will be destroyed after the event.
Prizes are divided into two parts, one part (A) rewarding the winners of each month and the other part (B) rewarding the winners of the whole year
(The following example is based on a $500,000 scenario, if it is 50,000 $MLN, it will be distributed in equal proportion)
A: Calculate the share price changes from the 1st of each month to the end of the month, the first place gets 10,000 US dollars, the second gets 5,000 US dollars, the third gets 2,000 US dollars, and the total of 12 months is 204,000 US dollars
B: Statistics of share price changes in 12 months, the first place is 100,000 US dollars, the second is 50,000 US dollars, the third is 20,000 US dollars, the 4th to 21st places get each 7,000 US dollars, and the total is 296,000 US dollars
**Duration**: One year (first round) (whether to carry out the second round depends on the effect of the first round)
**Necessary conditions for winners**: Contestants must first meet the following conditions before they can compete for prize money. If the number of finalists is less than the number of prize winners, the prize for the corresponding ranking will not be issued
1. The amount of the vault must be kept above USD $50,000 per day during the competition. This is to prevent the use of malicious means. For example, a manager creates multiple vaults and each vault double-down on one different coin. As long as there is a vault to obtain super high income, the manager can get a large bonus.
2. The performance of the vault in each settlement cycle must outperform half of the performance of BTC in the same period, e.g. BTC $16,000→$24,000 in Feb., (50%), the growth of vaults' share price must be ≥50%/2=25%, if BTC $16,000→$8,000, (-50%), the growth of vaults' share price must be ≥ -50%/2=-25%.
3. For the distribution of annual bonuses, there is an additional threshold, that is, regardless of the performance of BTC, the overall performance must be ≥ 20%.
 
**Other:**
1. During the competition period, half of the management Vault (creation, transaction) fee for all winners will be borne by the Enzyme agreement and repaid in $MLN at the end of the year. This part of $MLN is minted, and the rest will be destroyed after the event.
2. Monitor the trading behavior of each vault in real-time. If there is a high degree of similarity, the follower will be disqualified from the competition. This measure can prevent attacks such as copy-trading, and a single manager can create multiple vaults using the same trading strategy to obtain additional bonuses.
3. The best time to hold the event is in the early stage of the bull market. On the one hand, the manager’s investment performance is largely affected by the market cycle. On the other hand, in the bull market, activities such as marketing will get much higher returns than that in the bear market. But when the bull market will come is an uncertain thing. From the current point of view, starting the entire event in January 2024 seems to be a good choice. I suggest that we make these preparations first and wait for the best time.
 
**Star Managers Campaign-----**
1. Before the event starts, a certain budget needs to be prepared to promote the event to the whole crypto space and attract the best investment experts in this industry; the key points of publicity are:
a. Enzyme is a safe, non-custodial asset management platform, whether it is a manager or an investor, there is no need to worry about funds being involved
b. Star managers will display their talents here, and they can expand their investment scale and establish their own funds without establishing a financial institution
c. There is no threshold for investors to invest here, no strict investment and exit conditions, and no need to rely on contacts and resources. It is difficult for them to obtain such high returns elsewhere.
d. High bonus: $100,000 US dollars for the champion, and up to 220,000 US dollars (12*10,000+100,000) for individuals.
2. After the performance of each month comes out, promote the star managers of the month, including but not limited to: personal pages, interviews, podcasts, feature articles, media reports, etc., Promote star managers to the market, let the market know them, and enhance their trust and recognition
3. After the annual event, an intensive promotion will be carried out.
 
**Costs and expectations:**
1. Contest prize money is the main cost of the plan. US$500,000 is equivalent to 1% of the current $MLN FDV ($MLN@$27, FDV=US$56 million). It is expected that the plan can attract more than 50 vault managers (increase by 50/1200=4%), and (after removing the impact of token prices) drive at least 10 million US dollars in TVL (increase by 10m/52m=20%), in a bull market hopefully TVL to grow to $500 million (~900%)
2. If the current FDV/TVL ratio remains unchanged, i.e. $51,192,271/ $55,996,244=0.914, then based on the most conservative TVL growth estimate of $10 million, the new FDV is ($55,996,244+$10,000,000)*0.914=$60,320,567. In fact, the newly minted $MLN (at most 50,000 $MLN) will not dilute the total market value of Enzyme. On the contrary, the value of each $MLN will increase to $60,320,567/(2,069,711 +50,000)=$28.45. This is only based on the most conservative $10 million TVL growth scenario. With $100M of TVL growth, the value per $MLN grows to $67.
3. The real benefit to Enzyme is not just the growth in numbers, but letting more people know about Enzyme, educating users in the market, and also bringing Enzyme a group of outstanding star managers. More star managers → more vault with great performance → more investors and TVL → managers get a larger investment base, more profits → attract more star managers to settle in → ……. This is Enzyme's flywheel!
 
**Risks and concerns:**
1. The operation effect is not as good as expected, there are not enough good star managers to settle in, there is no vault with a high return rate, and it is impossible to further attract investors and TVL. The 500,000 US dollars and marketing expenses are diluted into the current $MLN FDV (1% dilution ratio).
2. The bull market is delayed, or its trend is difficult to figure out, which leads to the overall poor performance of vault managers. 
 
Generally speaking, I think this is a very worthwhile approach. **At the cost of about 1% FDV dilution, exchange for the possibility of Enzyme Fly Wheel.** Even if it is unsuccessful, at least let the users of the entire crypto space know about Enzyme. I always firmly believe that it is only a matter of time before this project, which has been in serious operation for more than 6 years, becomes the leader of DEFI with a TVL of 1 billion US dollars. The star manager program is to make this day come earlier (by 1~2 super cycles).
 
I hope you evaluate the program carefully. I'm open to any discussion, suggestions, and criticism.
 (The idea is inspired by my friend Hua L.)
Contact: jindounzsw@gmail.com, Twitter: @jindouyunz

**Comment by jindouyunz** [comment]
I'm not a native English speaker, sorry for the unclear, grammatical issues.

**Comment by Chris2pherChase** [comment]
Hey @jindouyunz just an FYI we on the Council are reviewing this proposal. Thank you for writing it up

**Comment by jindouyunz** [comment]
> Hey @jindouyunz just an FYI we on the Council are reviewing this proposal. Thank you for writing it up

Thank you! Really appreciate it!

**Comment by perhamgirl** [comment]
Thanks for this proposal . I found it super interesting to read through and think about. We have discussed a similar idea in the past at a Council level but never really taken it through to execution so I’m glad you brought it up again!

We have a few questions;

- Do you think managers who participate in this competition should be known parties and enter some kind of application process?
- If we are selecting winners, we have to establish categories and benchmarks. Eg. best performer against dollar or ETH or BTC? What are you thinking here?
- Linked to the above question, maybe it would be better to have a USD-based category and an ETH-denominated category? This would require a slightly larger pool but potentially could be offset by paying out prizes every quarter instead of every month?


We also have some comments:

- We suspect that we won’t need to mint new tokens for this program. At current prices it would require about 23K MLN per year which is affordable imo. This amount could be pre-allocated to the program each year and released if not spent. If the price of MLN appreciates, the amount of MLN tokens allocated could also be substantially reduced over time.
- It's possible that evaluating the performance of all candidates every month might be very operationally draining. Mabe having quarter and annual prizes instead would be less operationally burdensome.
I think that condition 2+3 needs to be adapted depending on which currency denomination we are evaluating based on.
- Not super big on sponsoring set up and half of the transaction fees unless gas fees become a serious issue again. But if we do, we should have a MLN-denominated cap on this.
We’re not too concerned about copy-trading vaults because by definition their performance will always be a bit worse than the person they’re copying . However ,I do think there are other things that need to be monitored and that argues for quarterly award events rather than monthly in order to reduce the reporting & assessment burden. Ideally we should encourage that managers do this with their own money initially to show skin in the game and to build somewhat of a track record.


In short, we think it's hard to know what kind of time and effort this will take from us operationally, so maybe we can start with a soft-launch experiment for one quarter and see how many people apply to participate and how easy or difficult it is to monitor the program. Ideally building some screener tools that can spot disqualifiable behaviour is helpful so that we can automate the scanning process.

**Comment by jindouyunz** [comment]
I am very happy to receive your reply! Here are some of my thoughts.

>Do you think managers who participate in this competition should be known parties and enter some kind of application process?

No need to be known parties and enter an application. There are three reasons for this. 
Firstly, we aim to select excellent investment managers, and long-term performance is the best proof. Whether or not they are known parties, as long as they achieve outstanding results on the Enzyme platform in the end, they will be excellent investment managers. 
The second reason is that in my understanding of the crypto field, celebrities are usually individual KOLs or VC investors, and there are few who become famous as secondary-market traders. Even those who claim to be masters of secondary-market trading with over 100k followers on Twitter may be a product of propaganda; again, performance is the best proof. 
Another reason for doing this is to keep Enzyme open and permissionless which is also one of DeFi's core values.

>  If we are selecting winners, we have to establish categories and benchmarks. Eg. best performer against dollar or ETH or BTC? What are you thinking here?

I prefer against USD. Three reasons: 
firstly, although I have learned that some DEGEN people always maintain ETH or BTC standards, the vast majority of industry investors still measure their asset changes in USD; 
secondly, this conforms to the common practice of the mainstream finance and investment industry; 
thirdly, at present, BTC and ETH still fluctuate greatly during bull and bear cycles. For example, 1,000 ETH can be either 1.2 million USD or 4.8 million USD. There is a huge difference between them.
In addition, regarding the selection of winners as mentioned in the above article, winners should be decided based on the share price growth rate (%) after meeting those necessary conditions.

>Linked to the above question, maybe it would be better to have a USD-based category and an ETH-denominated category? This would require a slightly larger pool but potentially could be offset by paying out prizes every quarter instead of every month?

USD-based category, the reason is the same as before. I think setting up a bonus pool on a quarterly basis is also possible. The reason for considering monthly distribution before was that it would provide more timely feedback to participants/investment managers and generate more stickiness (i.e. delayed gratification may cause participant loss). Of course, you can choose the quarterly basis if it can significantly lower the operation cost.

>We suspect that we won’t need to mint new tokens for this program. At current prices, it would require about 23K MLN per year which is affordable imo. This amount could be pre-allocated to the program each year and released if not spent. If the price of MLN appreciates, the amount of MLN tokens allocated could also be substantially reduced over time.

Great！ By the way, 23K $MLN（or $500,000)  is just a suggested value. In fact, the larger the value, the more generous the bonus and the better the promotion effect it will have. It can attract more skilled investors to participate in the competition. You can think of it as another form of liquidity mining. Most defi tokens used for LM are not less than 10%, and some even reach 60%. Illuvium, as a relatively well-known crypto game, plans to use 10% of its total token supply as a prize pool for competitions to attract players. The value I proposed is actually very small in proportion. If possible, I hope that there will be more bonuses within your acceptable range. For example, $1 million USD?

>It's possible that evaluating the performance of all candidates every month might be very operationally draining. Mabe having quarter and annual prizes instead would be less operationally burdensome.
>I think that condition 2+3 needs to be adapted depending on which currency denomination we are evaluating based on.

Indeed, this is an energy-consuming matter. Considering operating costs, you can change the monthly competition to a quarterly one.

>Not super big on sponsoring set up and half of the transaction fees unless gas fees become a serious issue again. But if we do, we should have a MLN-denominated cap on this.

Agree

>We’re not too concerned about copy-trading vaults because by definition their performance will always be a bit worse than the person they’re copying . However ,I do think there are other things that need to be monitored and that argues for quarterly award events rather than monthly in order to reduce the reporting & assessment burden. Ideally we should encourage that managers do this with their own money initially to show skin in the game and to build somewhat of a track record.

Agree
 
> In short, we think it's hard to know what kind of time and effort this will take from us operationally, so maybe we can start with a soft-launch experiment for one quarter and see how many people apply to participate and how easy or difficult it is to monitor the program. Ideally building some screener tools that can spot disqualifiable behaviour is helpful so that we can automate the scanning process.

Agreed, we can try operating for a quarter and see how it goes. Some thoughts:
1. Regarding how to select winners, there is actually no need to do additional development work. It can be done entirely on the basis of Enzyme's current capabilities. Choose each Vault's **share price** as an indicator, take a snapshot at the beginning and end of each phase of the competition, calculate the growth rate, and those with higher increases will be considered winners.
2. Even if it is just a trial operation, I suggest doing it during the early stages of a bull market because market conditions are crucial for the Contest's success. With the same cost and effort invested, much greater returns can be achieved during a bull market period. Moreover, events like this trading competition can only truly test their feasibility during bullish times since most traders find it difficult to generate profits during bear markets; many successful investors wait patiently for bull markets.
3. The details of the proposal (such as prize amounts, number of winners, allocation rules, etc.) are open for discussion.

**Comment by jindouyunz** [comment]
Hi Enzyme Team and council. I sense the new cycle is coming in 2024. Q1 is a quite good time to launch this program. 
How do you think about it?

**Comment by jindouyunz** [comment]
There are two things preventing Enzyme to become a 1 Billion TVL protocol:
1. The asset universe is limited due to Chainlink oracles: Even as a fan of Enzyme, I have to temporarily withdraw money from Enzyme to buy what I want to buy.
2. There aren't many good investors/traders on Enzyme: They are in the market, playing their own game, without knowing that they could leverage their talent via Enzyme. This is the purpose of this Star Manager Program, to incentivize them to come here.

**Comment by Chris2pherChase** [comment]
@jindouyunz let's DM so we can set up a call and explore this more. you can reach me in discord or telegram @ Chris2pherChase on both

**Comment by jindouyunz** [comment]
> @jindouyunz let's DM so we can set up a call and explore this more. you can reach me in discord or telegram @ Chris2pherChase on both

Happy to hear this! I sent you a message in Discord, please check it.

**Comment by jindouyunz** [comment]
Hi Enzyme Team and Council,
We're approaching 2024, and already in a new cycle. I think it's the best time to launch this Star Manager Program, at least, to give it a try. Passively waiting won't help Enzyme.
